app/models/manageiq/providers/workflows/automation_manager/workflow.rb

```diff
@@ -1,4 +1,28 @@
 class ManageIQ::Providers::Workflows::AutomationManager::Workflow < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
+  def self.seed
+    manager = ManageIQ::Providers::Workflows::AutomationManager.in_my_region.first
+    return if manager.nil?
+
+    repo = manager.configuration_script_sources.find_by(:name => ManageIQ::Providers::Workflows::AutomationManager::ConfigurationScriptSource::BUILTIN_REPO_NAME)
+    return if repo.nil?
+
+    to_delete = repo.configuration_script_payloads.pluck(:id)
+    Vmdb::Plugins.embedded_workflows_content.each do |filename|
+      plugin, path = filename.to_s
+                             .match(/.+\/(?<plugin>.+)\/content\/workflows\/(?<path>.+)/)
+                             .values_at("plugin", "path")
+
+      payload  = File.read(filename)
+      workflow = repo.configuration_script_payloads.find_or_initialize_by(:name => "#{plugin}/#{path}")
```

Comment:
NOTE we use `:name` as the unique attribute and set it to the file path to be consistent with how embedded ansible was behaving.

Now I'm wondering if we'd be better off setting `manager_ref` to the filename, treating that as unique, and setting `name` to the `"Comment"` value like we do in create_from_json below.  Would be a simple db migration to set `manager_ref = name` for all "embedded" configuration_scripts

Reply:
This can be a follow-up not directly related to this